packages/driver/src/cy/commands/window.ts

```diff
@@ -35,6 +36,24 @@ type CurrentViewport = Pick<Cypress.Config, 'viewportWidth' | 'viewportHeight'>
 // refresh would cause viewport to hang
 let currentViewport: CurrentViewport | null = null
 
+interface InternalTitleOptions extends Partial<Cypress.Loggable & Cypress.Timeoutable> {
+  _log?: Log
+}
+
+interface InternalWindowOptions extends Partial<Cypress.Loggable & Cypress.Timeoutable> {
+  _log?: Log
+  error?: any
+}
+
+interface InternalDocumentOptions extends Partial<Cypress.Loggable & Cypress.Timeoutable> {
```

Comment:
Can we reuse this interface?

Reply:
I've chose this way because they're for different commands. They can grow differently. And it's only 2 of them (it's not 3 strike out yet.)